webpack/assets/javascripts/react_app/components/HostDetails/Tabs/index.js

```diff
@@ -33,4 +34,13 @@ export const registerCoreTabs = ({ except = [] }) => {
       title: __('Reports'),
     }
   );
+  addGlobalFill(
+    TABS_SLOT_ID,
+    'Parameters',
+    <ParametersTab key="host-details-parameters-tab" />,
+    1000,
```

Comment:
On the original mockups Parameters is shown between Content and Traces. So I think the weight should be around 850.